### ISSUE TITLE
Add loadBalancerClass support

### ISSUE THREAD

**Issue by stevefan1999-personal** [openelb/openelb#367]
Since loadBalancerClass is [already stable in v1.24](https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class), why shouldn't we implement this?

**Comment by chaunceyjiang** [comment]
> Since loadBalancerClass is [already stable in v1.24](https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class),

Yes, I'm also very looking forward to this feature. Could you help implement this feature? If you encounter any problems, I would be very happy to help.

**Comment by stevefan1999-personal** [comment]
@chaunceyjiang right, so what do you think about just using `openelb`, or like `openelb-bgp`, `openelb-arp` for the class name?

**Comment by chaunceyjiang** [comment]
> using openelb 

+1
